### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by donavanbecker** [homebridge-plugins/homebridge-august#93] [Announcements]
### What's Changes
- Moved from CommonJS to ES Module
- Housekeeping and updated dependencies.

**Full Changelog**: https://github.com/donavanbecker/homebridge-august/compare/v1.3.4...v2.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/donavanbecker/homebridge-august/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by gadgetstar** [comment]
Hi, Since Yale today announced that they are discontinuing the Yale Conexis L1 app in favour of Yale Home, I have been advised that you need the Yale Access Module to use the new app. I have been using the Z-Module 2 to connect to my z-wave HA system and then Homebridge to expose the locks to Apple Home. Everything works perfect. 

If I have the replace the z-wave modules with Yale Access Modules to use the app, then I will need the Yale Connect Bridge to allow remote access (through the app). I want to retain the locks being updated and controllable in Apple Home. 

Does this plugin work with locks that are connected to Yale Connect Bridge or only the locks listed?

I have the Conexis L1 locks.

thanks.

**Reply by townsmcp** [reply]
Check out the table on the following page:

https://yalehome.co.uk/yale-module-for-your-smart-lock

I have a Conexis L2 (which comes with the Access BLE module and WiFi bridge) which I have on the Yale Home ap
. In the last couple of days I have bought the ZWave module (which I haven’t fitted yet) and will pair with SmartThings initially and if that goes well will purchase a zwave stick for Home Assistant. I have had so many issues with the BLE and WiFi side of things for a long time hence the switch. WiFi bridge is just far to temperamental - HA sometimes gets updated on lock status but only if I push a button in HA to get refresh status. My bridge is about 1m away from the lock at most. I have a WiFi access point in the bedroom directly above the Yale bridge so connectivity isn’t an issue. I also have an Everything Presence One right next to the access point and I have ESP Home with bluetooth proxy running. This is more reliable than the WiFi bridge (I get locked status update fairly often but no unlock update). 

Can’t you push HA to HomeKit?

**Reply by gadgetstar** [reply]
Maybe I wasn't clear in my original post. My current set up is a z-wave module 2 in the Conexis lock which connects to a  Hubitat C8, A Raspberry Pi running Homebridge then provides the link to Homekit so the locks are controllable via Apple Home. This works really well.

Currently with the Conexis App you can add or remove and individual cards/tag should you need to. My understanding is that you will only be able to do this with the new Yale Home app if you use their Access Module, which will mean you cannot use the z-wave module.

I wondered whether it was possible to use Homebridge with the Yale Connect WIFI bridge so the locks will still be controllable via Apple Home, and I can still add/remove individual tags/cards. Your comment that the Wifi bridge is not reliable makes me think that the best thing to do is to forego the Yale apps and stick with the z-wave module.

If you do not use the Yale apps then you have to manually add/remove tags in bulk. To add or delete 1 you need all of your tags handy as you need to add them all again. PITA but better than an unreliable bridge.

**Reply by gadgetstar** [reply]
Update: 
I received a Yale Home module. Removed the z-wave module and reset the lock. I then inserted the Yale Home module and configured it with the Yale Home app. The app then updated the locks firmware. This was a blessing as it updates the outdoor sensor so the rain doesn’t keep setting it off (no more annoying beeps). 

Once updated I then added the users and the tags/cards one by one. Once all the tags/cards were added I locked the door and then removed the Yale Home module and inserted the z-wave module.
Note: you need to remove a battery before removing or reinserting a module.

The lock was back on the z-wave network, has new firmware, and has the tags/cards working.

If I ever need to remove or add a card/tag I swap the modules and use the Yale Home app. Once done I make sure I lock the door and swap the modules back.

If you want to migrate completely to the Yale Home app but still see the locks in HomeKit I used the Yale Connect WiFi Bridge and connected this to the SmartThings app and then used the SmartThings plugin in HomeBridge. The locks work perfectly but you are using a cloud service rather a completely on-prem solution. Since I set this up to test, this is why I always made sure the lock was locked before removing the Yale Home module, otherwise SmartThings keeps telling you lock is unlocked. 

I couldn’t get the locks to work with the August plugin.

**Reply by townsmcp** [reply]
When using the Yale Connect Wifi Bridge, was you seeing correct lock status when unlocking the door manually rather than via automation/app? If I initiate unlock via app the status is correct however when someone unlocks the door by the lock itself the only entity that updates for me is the DoorSense when the door is opened - the lock status still always shows as locked :(

**Reply by gadgetstar** [reply]
I am sure it did, but I now have the z-wave module installed and both Hubitat and HomeKit report the correct status regardless of whether the lock is unlocked by the knob, tag, or app.

**Reply by townsmcp** [reply]
One of your comments above says ' have been using the Z-Module 2 to connect to my z-wave HA system' - can I confirm, you dont mean Home Assistant on this point? I have an old Hubitat hub knocking around somewhere (think it is a C4 where zwave and zigbee come on seperate devices) as well as the Smarthings hub - both of which are offline at the moment as I run all through Home Assistant (which I have configured to push to HomeKit) - just trying to find the best way forward so I can get the correct status at all times of the lock - I even have ESPHome devices near the door to pick up BLE for lock status which is sent into HA - again this works if locked/unlocked by an app but never hy the knob

**Reply by gadgetstar** [reply]
Hubitat C8 (I’m on my 4th gen Home Automation hub)
